README.md

```diff
@@ -72,12 +72,11 @@ And the corresponding "date" function that returns a date for
 each data point.
 
 ```js
-var eventDropsChart = d3.chart.eventDrops();
+var eventDropsChart = d3.chart.eventDrops()
+                              .date(d => d.date);
```

Comment:
@JulianoEngineer : I would simply add 4 spaces. It is too far away here. :)

```
var eventDropsChart = d3.chart.eventDrops()
    .date(d => d.date);
```

@Kmaschta: my bad. Seems that GitHub replaced several spaces by a single one. @sedy-bot, I beg your pardon!